include/lc3.h

```diff
@@ -145,9 +145,18 @@ extern "C" {
  *   LC3_CHECK_SR_HZ(sr)  True when samplerate in Hz is suitable
  */
 
+#if defined (INCLUDE_2M5) || defined(INCLUDE_05M)
```

Comment:
Can you limit the compile checks to INCLUDE_2M5 and INCLUDE_05M to the large generated tables only ?
In most instances the code would gain in legibility by removing those, and not much is gained (memory, code size) by having them.

Reply:
If library was built without INCLUDE_2M5 and INCLUDE_05M, code using it will fail at compile-time rather than at runtime due to missing tables.

Reply:
I understand, but can you try to minimize the number of checks required ?

Reply:
Sure, some checks (which do not use lc3_dt values) were removed in 1f48bc4

Reply:
Removed in 4b98fe7